sdk/identity/azure-identity/test/simplified_header.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief makes sure azure/identity.hpp can be included.
+ *
+ * @remark This file will catch any issue while trying to use/include the identity.hpp header
+ *
+ */
+
+#include <azure/identity.hpp>
+#include <gtest/gtest.h>
+
+class DllExportTest {
+  AZ_IDENTITY_DLLEXPORT static const bool DllExportHIncluded;
+};
+
```

Comment:
Why is this needed?

Reply:
This is to verify that `dll_import_export.hpp` is included (=macro `AZ_IDENTITY_DLLEXPORT` is defined and can syntactically be used in the context it is supposed to be used).